exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/FilterEvaluatorUtils.java

```diff
@@ -54,9 +54,12 @@
   private FilterEvaluatorUtils() {
   }
 
+  @SuppressWarnings("unchecked")
   public static RowsMatch evalFilter(LogicalExpression expr, MetadataBase.ParquetTableMetadataBase footer,
                                      int rowGroupIndex, OptionManager options, FragmentContext fragmentContext) {
-    List<SchemaPath> schemaPathsInExpr = new ArrayList<>(expr.accept(new FieldReferenceFinder(), null));
+    //This change is necessary to prevent build errors on JDK 1.8.0_65.  See (https://bugs.openjdk.java.net/browse/JDK-8066974)
```

Comment:
Please rephrase it to:
```suggestion
    // Specifies type arguments explicitly to avoid compilation error caused by JDK-8066974
```